africanus/calibration/utils/correct_vis.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+from africanus.util.docs import DocstringTemplate
+from africanus.util.numba import generated_jit, njit
+from .utils import check_type
```

Comment:
Use absolute rather than relative import

```suggestion
from africanus.calibration.utils import check_type
```

Reply:
Ah man. This was such a neat solution for my awkwardly broken lines when they get too long. Any chance of relaxing the strict 79 character limit?